charts/redpanda/templates/connectors/connectors.yaml

```diff
@@ -50,12 +50,18 @@ limitations under the License.
   {{ $consoleSASLConfig = cat $consoleSASLConfig " echo $CONNECT_SASL_PASSWORD > /opt/kafka/connect-password/rc-credentials/password;" }}
   {{ $consoleSASLConfig = cat $consoleSASLConfig " exec /opt/kafka/bin/kafka_connect_run.sh" }}
   {{ $command = append $command $consoleSASLConfig }}
+
+  {{ $extraVolumes = concat $extraVolumes .Values.connectors.storage.volume }}
```

Comment:
I'm not sure why this `concat` is added in this PR. How that relates to audit logging?

Reply:
It was added in https://github.com/redpanda-data/helm-charts/pull/941. I don't understand why Github shows this change..